docs/core/whats-new/dotnet-9/libraries.md

```diff
@@ -493,9 +584,47 @@ For example, `String.Join` now includes the following overload, which implements
+
+In .NET 8, a set of `Split` and `SplitAny` methods were introduced for `ReadOnlySpan<char>`. Rather than returning a new `string[]`, these methods accept a destination `Span<Range>` into which the bounding indices for each component are written. This makes the operation fully allocation-free. These methods are appropriate to use when the number of ranges is both known and small.
+
+In .NET 9, new overloads of `Split` and `SplitAny` have been added to allow incrementally parsing a `ReadOnlySpan<T>` with an *a priori* unknown number of segments. The new methods enable enumerating through each segment, which is similarly represented as a `Range` that can be used to slice into the original span.
+
+```csharp
+public static bool ListContainsItem(ReadOnlySpan<char> span, string item)
+{
+    foreach (Range segment in span.Split(','))
+    {
+        if (span[segment].SequenceEquals(item))
+        {
+            return true;
+        }
+    }
 
-The following changes have been made in the <xref:System.Numerics> namespace:
+    return false;
+}
+```
+
+## System.Guid
+
+<xref:System.Guid.NewGuid> creates a `Guid` filled mostly with cryptographically secure random data, following the UUID Version 4 specification in RFC 9562. That same RFC also defines other versions, including Version 7, which "features a time-ordered value field derived from the widely implemented and well-known Unix Epoch timestamp source". In other words, much of the data is still random, but some of it is reserved for data based on a timestamp, which enables these values to have a natural sort order. In .NET 9, you can create a `Guid` according to Version 7 via the new `Guid.CreateVersion7()` <!--<xref:System.Guid.CreateVersion7?displayProperty=nameWithType>--> and `Guid.CreateVersion7(DateTimeOffset timestamp)`  <!--<xref:System.Guid.CreateVersion7(System.DateTimeOffset)?displayProperty=nameWithType>--> methods. You can also use the new `Version` <!--<xref:System.Guid.Version>--> property to retrieve a `Guid` object's version field.
+
+## System.IO
+
+- [Compression](#compression)
+- [XPS documents from XPS virtual printer](#xps-documents-from-xps-virtual-printer)
+
+### Compression
+
+<xref:System.IO.Compression> features like <xref:System.IO.Compression.ZipArchive>, <xref:System.IO.Compression.DeflateStream>, <xref:System.IO.Compression.GZipStream>, and <xref:System.IO.Compression.ZLibStream> are all based primarily on the zlib library. Starting in .NET 9, these features instead all use zlib-ng, a library that yields more consistent and efficient processing across a wider array of operating systems and hardware.
+
+### XPS documents from XPS virtual printer
+
+XPS documents coming from a V4 XPS virtual printer previously couldn't be opened using the <xref:System.IO.Packaging> library, due to lack of support for handling *.piece* files. This gap has been addressed in .NET 9.
```

Comment:
> *.piece*

I would have expected a filename to be use the `code-fence` format. Is that a style guide preference?

Reply:
Yes, it comes from here: https://review.learn.microsoft.com/en-us/help/contribute/text-formatting-guidelines?branch=main&branchFallbackFrom=master#italics